src/promptflow/promptflow/_sdk/_service/apis/collector.py

```diff
@@ -28,7 +29,15 @@
 from promptflow._utils.thread_utils import ThreadWithContextVars
 
 
-def trace_collector(logger: logging.Logger):
+def trace_collector(get_created_by_info_with_cache: Callable, logger: logging.Logger):
+    """
```

Comment:
may refer to our doc on docstring to align the format

Reply:
Will improve after confirming about how to reuse this part of code.